### DISCUSSION TITLE
v4.28版本windows安全中心报毒

### DISCUSSION THREAD

**Discussion by nanmingxwx** [MaaAssistantArknights/MaaAssistantArknights#7647] [Q&A]
v4.28版本windows安全中心报毒

**Comment by nanmingxwx** [comment]
![1702699056423](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/153969945/78d6e251-0650-488f-a414-55e936212103)

**Comment by Alerwi** [comment]
i have same issue, and it happens after my OS updated to Win11 22H2.
and MAA stable version(upedated today) also shares this isuue.

**Reply by Alerwi** [reply]
it works after add MAA root path into Windows Defender White List ...

**Comment by ABA2396** [comment]
#6662
